data/human/marauders.txt

```diff
@@ -169,12 +169,12 @@ ship "Marauder Bounder" "Marauder Bounder (Engines)"
 	add attributes
 		"engine capacity" 15
 	outfits
-		"Heavy Laser"
 		"Quad Blaster Turret" 2
 		
 		"Dwarf Core"
 		"LP072a Battery Pack"
-		"S-270 Regenerator"
+		"Supercapacitor" 3
```

Comment:
I feel like a blaster to fill the gun slot emptied by the heavy laser would be better here than stacked supercaps when it already has another battery.

Reply:
I prefer a heavy rocket pod

Reply:
Brutal. 😁